susemanager/susemanager.spec

```diff
@@ -25,19 +25,21 @@
 %if 0%{?rhel}
 %global apache_user root
 %global apache_group root
-%global tftp_group root
+%global pgconfig /var/lib/pgsql/data/postgresql.conf
```

Comment:
does this really exists when there is no database initialized?
For SUSE we use `/etc/sysconfig/postgresql` to specify some values which define how the DB is started (commandline options) and in which directory the DB should store the files (e.g. /var/lib/pgsql/data/).
One of the values is the language and encoding. 

On a fresh installed SUSE system `/var/lib/pgsql/data/postgresql.conf` does not exist. Only after postgresql was started for the first time.
Is this different on RHEL?

Reply:
hmm, you are right. It will not exist and created by spacewalk-setup. Seems I misunderstood the purpose of `/etc/sysconfig/postgresql`. Let me see if there is something similar on EL.

Reply:
@mcalmer I just cross checked and couldn't find a similar mechanism on AlmaLinux. Can we remove the whole  %posttrans section and put the logic into spacewalk-setup? This is where we create the DB in the first place.

Reply:
I think this section was made to "fix" existing DB configurations. I would suggest to simply write a 

```
%if 0%{suse_version}
...
%endif
```
macro around the whole %posttrans section.
We can also think about to remove this completely as I think we changed the setup to initialize always with `en_US.UTF8`.
In the meanwhile all customers should have this.
But we should double check if we really initialize correctly now.

Reply:
ok, then I will

1. put an if statement and a comment around the posttrans section.
2. Verify the setup script.
And then resubmit.

Reply:
I have now moved the `%posttrans` section from the spec file to `bin/mgr-setup` and verified the code in `bin/pg-migrate-x-to-y.sh`
Rationale:
The involved variable `POSTGRES_LANG `is only used during database initalisation. So the changes in `%posttrans` would only be relevant for new databases/upgrades. The Uyuni setup now checks for the right locale; the upgrade script did check for the right locale setting already.